src/udiskslinuxfilesystem.c

```diff
@@ -558,7 +559,7 @@ add_acl (const gchar  *path,
       udisks_warning(
                    "Adding read ACL for uid %d to `%s' failed: %m",
                    (gint) uid, path);
-      chown(path, uid, -1);
+      chown(path, -1, gid);
```

Comment:
Yes, of course, good point.  I did it as a separate commit that's overwritten by the subsequent main commit, so that `git blame` still shows the real change.  We can also squash them together, if you want.